backend/core/src/user-applications/user-applications.controller.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  Body,
+  ClassSerializerInterceptor,
+  Controller,
+  Delete,
+  Get,
+  Param,
+  Post,
+  Put, UseGuards,
+  UseInterceptors
+} from "@nestjs/common"
+import { UserApplicationsService } from "./user-applications.service"
+import { ApplicationCreateDto } from "./application.create.dto"
+import { ApplicationUpdateDto } from "./application.update.dto"
+import { ApplicationDto } from "../applications/applications.dto"
+import { JwtAuthGuard } from "../auth/jwt.guard"
+import { OwnerGuard } from "../auth/owner.guard"
+import { ApiBearerAuth } from "@nestjs/swagger"
+
+@Controller("user")
+@UseGuards(JwtAuthGuard)
+@ApiBearerAuth()
```

Comment:
Isn't this duplicating the JWT logic from the Auth guard? It checks for a `Bearer token` too...

Reply:
It is not. It only adds metadata for swagger generation. See 
https://github.com/nestjs/swagger/blob/master/lib/decorators/api-bearer.decorator.ts and https://github.com/nestjs/swagger/blob/master/lib/decorators/api-security.decorator.ts